developing/proxstar-postgres/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM postgres:12
+
+RUN apt-get update \
+    && apt-get install -y postgresql-plperl-12 postgresql-plpython3-12 libnet-ip-perl libnet-ldap-perl libnet-dns-perl libnet-snmp-perl libnet-server-mail-perl libcrypt-des-perl build-essential cpanminus \
+    && rm -rf /var/lib/apt/lists/* \
+    && cpanm Data::Validate::Domain \
+    && apt-get remove -y build-essential
+
+#COPY ./schema/large.sql /docker-entrypoint-initdb.d/ # Woa there, pal; That's a lot of   S C H E M A.
+# I volunteer as tribute. DM me if this breaks.
+ADD https://csh.rit.edu/~wilnil/proxstar-postgres/large.sql /docker-entrypoint-initdb.d/
```

Comment:
Do we know how this schema got created in this first place?
I really thing this should live in the repo...

Reply:
I created the schema. I can document how I did so if you'd like.

Also, @galenguyer @mxmeinhold One final show of hands, are y'all for (:+1: ) or against (:-1:) separating the schema from the repo?

It do be **16,000** lines of sql.

Reply:
To be fair: 16k lines of sequel will compress very well
Not that I think CSH is going to disappear any time soon, but I feel like the repo should have everything you need, and a user page feels like it's liable to be broken in 10 years. RIT has a bunch of missing member pages now, for example.

Reply:
Ah. This is STARRS schema. That's fair... I would say it's okay to leave it out of the repo, but if you do break it into it's own, you should probably include it.
Also you probably should give some instructions on generating that schema, just in case something changes